cmds/token-signer/httpServer/HttpServer.go

```diff
@@ -0,0 +1,56 @@
+package httpServer
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"token-signer/config"
+	"token-signer/tokenSigner"
+)
+
+type HttpServer struct {
+}
+
+type RequestParser interface {
+	ParseRequest(r *http.Request) (tokenSigner.CreateSignedTokenRequest, error)
+}
+
+func (h HttpServer) Serve(signer tokenSigner.TokenSigner) {
+
+	conf := config.GetGlobalConfig()
+	var parser RequestParser = HttpRequestParser{}
+
+	http.HandleFunc("/token", func(w http.ResponseWriter, r *http.Request) {
+		log.Println("Token requested with parameters")
+		log.Println(r)
+
+		if r.Method != "GET" {
+			http.Error(w, "Invalid request method", 405)
```

Comment:
What do you think about changing this magic numbers and messages in errors being thrown on this server for something more meaningful? We could try to use go's http package for better readability. 

Reference:
- https://pkg.go.dev/net/http#Error
- https://pkg.go.dev/net/http#pkg-constants

Reply:
Fixed in [7b249d0](https://github.com/dp-icea/dss/pull/2/commits/7b249d044316f35806470341e388d217a097b694)